src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.File.Create.cs

```diff
@@ -9,7 +9,7 @@
 namespace System.IO.Tests
 {
     public class File_Create_Tests : FileSystemWatcherTest
-    {
+    {       
```

Comment:
please revert changes here.

Reply:
I need help here, i don't see any change to revert on this file on my source(VS), can you tell me how to revert this(maybe on this specific case so i can learn how to do for future)?

Reply:
It's a whitespace only change. You need to configure your diff tool to show whitespace only changes. Github also showed them by default.

Reply:
In this case you can just revert this file. Then all looks good, thanks!

Reply:
sry, I didn't notice your response: `git checkout -- path/to/file`